src/families/tezos/bridge/js.ts

```diff
@@ -43,9 +43,7 @@ import api from "../api/tzkt";
 
 const receive = makeAccountBridgeReceive();
 
-const EXISTENTIAL_DEPOSIT = new BigNumber(
-  DEFAULT_STORAGE_LIMIT.ORIGINATION * 1000
-);
+const EXISTENTIAL_DEPOSIT = new BigNumber(275000);
```

Comment:
this was a mistake I did, ORIGINATION is not 275*

Reply:
I thought it was the same, didn't paid attention also